### DISCUSSION TITLE
셀피와 사진찍기 기능

### DISCUSSION THREAD

**Discussion by uniconnect147** [naverz/zepeto-studio-kor#1109] [5. Q&A]
#437 의 가이드대로 하는데 셀카 기능이 작동하지 않습니다.
현재 scen에 ZepetoScreenShotModule까지 넣어둔 상태고 IKPass가 활성화된 Animator를 도저히 찾을수가 없습니다.

일단 샘플 ZepetoScreenShotModule을 scen에 넣었더니 일반적인 사진찍기는 가능하나
셀카를 찍을땐 다음과 같이 셀카위치의 메인카메라로 보이는곳만 찍히며 캐릭터 셀카봉도 보이지 않습니다.
![image](https://user-images.githubusercontent.com/109579358/182254253-f3a5b02d-6fc7-4835-9add-702b398a594c.png)
그리고 배경을 눌렀을땐 화면과같이 png처럼 빈공간으로 나오는게 정상인가요??
![image](https://user-images.githubusercontent.com/109579358/182254322-f0b35ba0-b0ea-42bf-9e2c-1240227dfbde.png)

답변부탁드립니다~
오늘도 좋은하루되세요!

**Comment by zlmrzlo** [comment]
animator 속성 잘 보시면 layer에 속성 중 ikpass있는 것으로 알고 있어요
